Formula/linux-headers@5.15.rb

```diff
@@ -5,6 +5,11 @@ class LinuxHeadersAT515 < Formula
   sha256 "593ae3930cabde390aa6105faa614f1d10b0b7eda782680a120e345de6ed256f"
   license "GPL-2.0-only"
 
+  livecheck do
+    url "https://mirrors.edge.kernel.org/pub/linux/kernel/v5.x/"
+    regex(/href=.*?linux[._-]v?(5\.15(?:\.\d+)+)\.t/i)
```

Comment:
```suggestion
    url :homepage
    regex(/href=.*?linux[._-]v?(5\.15(?:\.\d+)*)\.t/i)
```

The directory listing page works but it's ~67 KB and will continue growing as new versions appear, whereas the homepage is ~3 KB and shouldn't substantially grow. Besides that, the homepage links to the same URL we use as `stable` in the strategy, so I tend to prefer that anyway.

Reply:
make sense 👍